CMakeLists.txt

```diff
@@ -13,9 +13,24 @@ endif()
 include(${MBED_ROOT}/tools/cmake/core.cmake)
 include(${MBED_ROOT}/tools/cmake/profile.cmake)
 
-add_library(mbed-core OBJECT)
-# TODO CMAKE: Replace mbed-os by a new one that will include mbed-core and mbed-rtos
-add_library(mbed-os ALIAS mbed-core)
+add_library(mbed-core INTERFACE)
+
+# Mbed OS = mbed core + mbed rtos
```

Comment:
```suggestion
```
That can be seen in code.

Reply:
I was thinking if we should do more. I'll remove for now both comments

Reply:
fixed